client/src/components/Card/Card.scss

```diff
@@ -0,0 +1,29 @@
+.card {
+  flex: 0 1 22rem;
+  transition: box-shadow 0.3s ease-in-out;
+  margin-bottom: 1rem;
+  padding: 0 0 4px;
+  height: 100%;
+  background-color: whitesmoke;
+  color: gray;
+  break-inside: avoid;
+  border-radius: 4px;
+
+  h1,
```

Comment:
Could this and line 18-22 be combined? Also, are you using prettier here? My prettier styles this file differently (kinda weird that h1 is on a different line from the rest of the h[x]. I wonder if we want to add pretter and linting pre-commit checks to the github CI. 

These are all nits

Reply:
I've added ` "editor.formatOnSave": true,` to my .vscode settings.json file. 

Added this issue https://github.com/waterthetrees/wtt_front/issues/654